prow/plugins/require-matching-label/require-matching-label.go

```diff
@@ -236,11 +244,23 @@ func handle(log *logrus.Entry, ghc githubClient, cp commentPruner, configs []plu
 			if cfg.MissingComment != "" {
 				msg := plugins.FormatSimpleResponse(e.author, cfg.MissingComment)
 				if err := ghc.CreateComment(e.org, e.repo, e.number, msg); err != nil {
-					log.WithError(err).Error("Failed to create comment.")
+					log.WithError(err).Error("Failed to create missing comment.")
 				}
 			}
 		}
 
+		if cfg.MatchingComment != "" {
+			if hasMatchingLabel {
+				msg := plugins.FormatSimpleResponse(e.author, cfg.MatchingComment)
+				if err := ghc.CreateComment(e.org, e.repo, e.number, msg); err != nil {
```

Comment:
We need to avoid creating a comment if one has already been created, otherwise this will spam the PR on unrelated changes.